apps/frontend/static_src/scss/components/nav-buttons.scss

```diff
@@ -43,5 +44,16 @@
     &__label {
         @include fs(xs);
         font-weight: 700;
+
+        @include media-query(small) {
+            &:first-child {
+                padding: 1px 1px 1px 50px;
+            }
+
+            &:nth-child(2) {
+                position: absolute;
+                padding: 10px 1px 10px 50px;
+            }
```

Comment:
I believe this is unnecessary absolute positioning, everything can be achieved using Flexbox, see above comment.

```suggestion
```